main.star

```diff
@@ -47,6 +49,12 @@ def run(plan, args):
     
     plugins_dir = env_vars.get("PLUGINS_DIR", DEFAULT_PLUGINS_DIRNAME)
 
+    # validate plugins
+    plugins_names = env_vars[ALLOW_LISTED_PLUGINS_ENV_VAR_KEY].split(',')
+    are_all_required_env_vars_set, missing_required_env_vars = plugins.areAllRequiredEnvVarsSet(env_vars, plugins_names)
+    if are_all_required_env_vars_set == False:
+        fail("Error while validating the required env var for plugins. The missing required env vars are '{}'".format(missing_required_env_vars))
```

Comment:
@leoporoli can we disable this VAR check if `__plugin_branch_to_use` & `__plugin_repo_to_use` are set?

Why? Otherwise the tests we have published to others will start failing as interpretation errorrs. We can put a comment on top saying why we're doing this.

Also maybe we could add a "__skip_env_vars" flag; then we roll out this flag to our CI tests and then we disable the check for `__plugin_branch_to_use` & `__plugin_repo_to_use`; so basically

if "__plugin_branch_to_use" in env_vars and `__plugin_repo_to_use` in env_vars:
      dont_fail_if_missing_env_vars()
elif "__skip_env_vars" in env_vars:
      dont_fail_if_missing_env_vars()
else:
     fail_missing_env_vars()